lib/modulesync/cli/thor.rb

```diff
@@ -8,7 +8,19 @@ module CLI
     # This class extends ::Thor class to
     # - exit with status code sets to `1` on Thor failure (e.g. missing required option)
     # - exit with status code sets to `1` when user calls `msync` (or a subcommand) without required arguments
+    # - show subcommands help using `msync subcommand --help`
     class Thor < ::Thor
+      def self.start(*args)
+        if (Thor::HELP_MAPPINGS & ARGV).any? && subcommands.grep(/^#{ARGV[0]}/).empty?
```

Comment:
Should this use `Regexp.escape()`?
```suggestion
        if (Thor::HELP_MAPPINGS & ARGV).any? && subcommands.grep(/^#{Regexp.escape(ARGV[0])}/).empty?
```

Or perhaps even better:
```suggestion
        if (Thor::HELP_MAPPINGS & ARGV).any? && subcommands.any? { |command| command.start_with?(ARGV[0]) }
```

Reply:
Applied (with fix: `none?` instead of `any?`)

Reply:
Oh yes, my bad. Thanks for not blindly following suggestions :)